<?xml version="1.0"?>
<launch>

	<node pkg="mdr_pickup_and_place" type="pickup_and_place_node" name="pickup" respawn="false" output="screen">
		<remap from="joint_states" to="/joint_states" />
		<remap from="get_ik" to="/compute_ik" />
		<!--
		<remap from="joint_trajectory_action" to="/arm_controller/joint_trajectory_action" />
		<remap from="is_cylindric_grasped" to="/sdh_controller/is_cylindric_grasped" />
		<remap from="grasp_posture_control" to="/mdr_manipulation/grasp_posture_control" />
		-->

		<!-- the distance of the pre-grasp position to the object position (value given in meters) -->
		<param name="pre_grasp_distance" value="0.60" />

		<!-- how far to stop grasping before the object position (value given in meters) -->
		<param name="grasp_distance" value="0.18" />

		<!-- the distance over the object after the grasp (value given in meters) -->
		<param name="post_grasp_height" value="0.06" />
	</node>

	<!--node pkg="mir_moveit_scene" type="arm_workspace_restricter_node" name="workspace_restricter" respawn="false" output="screen"/-->
	<!-- include file="$(find mir_moveit_scene)/ros/launch/arm_workspace_restricter.launch"/ -->
</launch>
